hugo/config/_default/params.yaml

```diff
@@ -0,0 +1,54 @@
+assembly_fields:
+  description:
+    - key: name
+      label: Assembly Name
+    - key: type
+      label: Assembly Type
+    - key: level
+      label: Assembly Level
+    - key: repr
+      label: Assembly Representation
+    - key: accession
+  stats:
+    - key: length
+      label: "Assembly length (Mbp)"
+    - key: gc
+      label: "GC %"
+    - key: contig
+      label: "Contig #"
+    - key: contig_n50
+      label: "Contig N50 (Mbp)"
+    - key: contig_l50
+    - key: contig_n90
+      label: "Contig N90 (Mbp)"
+    - key: contig_l90
+    - key: scaffold
+      label: "Scaffold #"
+    - key: scaffold_n50
+      label: "Scaffold N50 (Mbp)"
+    - key: scaffold_l50
+    - key: scaffold_n90
+      label: Scaffold N90 (Mbp)
+    - key: scaffold_l90
+    - key: scaffold_gt_10
+      label: "Scaffolds >= 10 kb"
+    - key: busco
+      label: BUSCO %
+  annotation:
+    - key: gene_cnt
+      label: "Gene #"
+    - key: transcript_cnt
+      label: "Transcript #"
+    - key: exons_avg
+      label: Avg exons per transcript
+    - key: gene_length_avg
+      label: Avg gene length (bp)
+    - key: transcript_length_avg
+      label: Avg transcript length (bp)
+    - key: exon_length_avg
+      label: Avg exon length (bp)
+    - key: intron_length_avg
+      label: Avg intron length (bp)
+    - key: busco
```

Comment:
@brinkdp Thanks for your remark regarding the need to customize assembly fields.

What do you think about specifying fields in the site configuration (which keeps it DRY), with the possibility to flag optional ones ? I also allowed to specify extra fields in the species page front-matter (but that is probably not necessary)

Reply:
Totally agree that is something to strive for. But let's keep it for a christmas PR maybe, as this one is already getting way out of control 😅